custom_components/wyzesense/binary_sensor.py

```diff
@@ -43,9 +44,17 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+def findDongle(): # wmm
+    df = subprocess.check_output(["ls", "-la", "/sys/class/hidraw"]).decode('utf-8')
+    for l in df.split('\n'):
+        if ("e024" in l and "1a86" in l) or ("E024" in l and "1A86" in l):
```

Comment:
I wonder if it would be better to do something like:

```python
ll = l.lower()
if ("e024" in ll and "1a86" in ll):
```

This way take out 2 extra searches in each string plus reduces the chance that someone updates one set, but not the other.